src/task_queue/nudges/user_event_nudge.py

```diff
@@ -16,71 +17,80 @@
 WEEKLY = "per_week"
 BI_WEEKLY = "biweekly"
 MONTHLY = "per_month"
-DAILY="per_day"
+DAILY = "per_day"
```

Comment:
when you make a tiny change but then reformat the whole file, it make it difficult to review.